### ISSUE TITLE
Format parameters in lambda

### ISSUE THREAD

**Issue by paul-dingemans** [pinterest/ktlint#1976]
After updating to 0.49.0 this is still not working as ktlint expects the formatting to look like this:

```kotlin
fun combine(a: Any, b: Any, c: (Any, Any) -> Unit) {
  c(a, b)
}

class Foo {
  fun foo() {
    combine(
      Any(),
      Any()
    ) { a,
              b -> // Unexpected indentation (8) (should be 14) (standard:indent)
      TODO()
    }
  }
}
```

_Originally posted by @eygraber in https://github.com/pinterest/ktlint/issues/1756#issuecomment-1520584883_

**Comment by paul-dingemans** [comment]
In `0.49.0` it works correctly for code style `ktlint_official`. With the other code styles the formatting is indeed incorrect.
